src/index.ts

```diff
@@ -102,7 +102,7 @@ export { extractFiles } from "./exchange-connector/exchangeDirectoryParser";
 export { RestApi } from "./exchange-connector/exchangeTypes";
 export { ramlToolLogger } from "./logger";
 
-export { model } from "amf-client-js";
+export { model, AMF, plugins } from "amf-client-js";
```

Comment:
These aren't names we control. Should they be exported as `amf` to avoid potential name collisions?